### DISCUSSION TITLE
How to save data somewher on POST method?

### DISCUSSION THREAD

**Discussion by EncouragedEntity** [mockoon/mockoon#1238] [Help] [answered]
I want to post some data and save it, so when I perform GET method, it will be returned to me. I want to simulate storage, I guess to save and get all data I want while using the API

**Comment by jcbenck** [comment]
I guess you can use Databuckets for it:
https://mockoon.com/docs/latest/data-buckets/overview/

**Reply by 255kb** [reply, marked as answer]
Yes, I would say the easiest approach is to use a data Bucket with a [CRUD route](https://mockoon.com/docs/latest/api-endpoints/crud-routes/).
